java/java.lsp.server/vscode/src/extension.ts

```diff
@@ -85,9 +88,8 @@ export function activate(context: ExtensionContext) {
         }
     });
 
-    let log = vscode.window.createOutputChannel("Java Language Server");
     log.appendLine("Launching Java Language Server");
-    vscode.window.showInformationMessage("Launching Java Language Server");
+    vscode.window.setStatusBarMessage("Launching Java Language Server", 2000);
```

Comment:
Great. I wasn't aware of this method. Probably added in newer version of VSCode.